provisions/roles/cleanup/jenkins/master/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Stop jenkins systemd service
+  sudo: yes
+  service: name=jenkins state=stopped
+
+- name: Remove contents under direcotry /var/lib/jenkins/
+  sudo: yes
+  shell: rm -rf /var/lib/jenkins/*
```

Comment:
IMO, we should not delete /var/lib/jenkins. this will reset all the settings and may be all the jobs will be rebuilt even if there is no change to them.